fabric-particles-v1/src/main/java/net/fabricmc/fabric/api/particles/FabricSpriteParticle.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.particles;
+
+import net.fabricmc.fabric.impl.particles.ParticleManagerHooks;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.particle.ParticleTextureSheet;
+import net.minecraft.client.particle.SpriteBillboardParticle;
+import net.minecraft.client.texture.SpriteAtlasTexture;
+import net.minecraft.util.Identifier;
+import net.minecraft.world.World;
+
+/**
+ * Base class for custom particles using Fabric's particle API.
+ * 
+ * @author swordglowsblue
```

Comment:
We generally don't include author credits in the API source.  Source authorship is tracked in git and over time will be intermingled within each module.

Reply:
Fair enough, I figured that would be the case but I'd rather remove it after than not put it in if I should have. I'll remove them.